roles/hosted_engine_setup/tasks/create_target_vm/03_hosted_engine_final_tasks.yml

```diff
@@ -184,7 +184,7 @@
     args:
       warn: false
   - name: Initialize metadata volume
-    command: fallocate -l 1G "{{ he_metadata_disk_path }}"
+    command: dd bs=1M count=10 oflag=direct if=/dev/zero of="{{ he_metadata_disk_path }}"
```

Comment:
Looks good - if we need only few MiB, there is no need to use fancy mechanism for zeroing, using dd is good.